loadtest/config.go

```diff
@@ -69,12 +70,15 @@ type UserControllerConfiguration struct {
 	//   UserControllerSimple - A simple version of a controller.
 	//   UserControllerSimulative - A more realistic controller.
 	Type userControllerType
-	// A rate multiplier that will affect the speed at which user actions are
+	// A distribution of rate multipliers that will affect the speed at which user actions are
 	// executed by the UserController.
 	// A Rate of < 1.0 will run actions at a faster pace.
 	// A Rate of 1.0 will run actions at the default pace.
 	// A Rate > 1.0 will run actions at a slower pace.
-	Rate float64
+	RatesDistribution []struct {
+		Rate       float64
+		Percentage float32
```

Comment:
Any reason to keep this float32 and the other as float64 ? I don't mind either ways. Just curious.

Reply:
I've noticed that. We can probably move everything to float32 since we clearly don't need high precision.

Reply:
Apparently `float64` is pretty much the default as far as Golang is concerned so might aswell move everything to it. Using `float32` would fill us of `float64()` casting with no real added benefit.